### ISSUE TITLE
Low DEX values starting 2020-07-21

### ISSUE THREAD

**Issue by dlksmc** [COVIDExposureIndices/COVIDExposureIndices#8]
The DEX and adjusted DEX values in [the most recent commit of county_dex.csv](https://github.com/COVIDExposureIndices/COVIDExposureIndices/blob/1c7588c8f77ae638d37cdf76048a3e31ef5ba250/dex_data/county_dex.csv) have very low values for the last eight days (2020-07-21 through 2020-07-28).  The prior commit--which goes through 2020-07-27--is similar.

Refer to the plot below, which shows the daily average of `dex_a` for the US:

![dex_a](https://user-images.githubusercontent.com/15652271/89350327-d3bf3100-d67d-11ea-8d73-f092fa4e552b.png)

The plot below shows that nearly all counties have a `dex_a` value of 80 or lower starting 2020-07-21:

![dex_a_pct_lt_80](https://user-images.githubusercontent.com/15652271/89351161-6b714f00-d67f-11ea-905d-22409e63a2ad.png)

It looks like there is an issue with or change in the data, as  I would not expect the entire country to experience such a large decline at once.  Any ideas what's causing it?

Thank you for making this data available--it is very helpful!

**Comment by jdingel** [comment]
Thanks, Dan. We'll look into this.

**Comment by jdingel** [comment]
Dan, thank you for flagging this issue. When we revised some upstream code, we introduced a bug that substantially reduced the set of locations and devices used to calculate the DEX. That caused the clear break you flagged. We've fixed the bug and the series now looks sensible (see below for quick-and-dirty graph of average state-level DEX values relative to same day of week during first week of March). The correct values were published by commit a61678608d2e12095dee8417339f117a7f65edc9.

![image](https://user-images.githubusercontent.com/30936514/89482847-5f10f300-d760-11ea-9ccf-e3b7d1906e11.png)
